linkis-dist/package/db/upgrade/1.3.2_schema/mysql/linkis_Index_optimization_dll.sql

```diff
@@ -0,0 +1,159 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+
+
+ALTER TABLE `linkis_ps_configuration_key_engine_relation`
+        DROP INDEX IF EXISTS `config_key_id`,
```

Comment:
file name `linkis_ddl.sql`

Reply:
Some indexes add table name abbreviations because some indexes have the same name, so table name abbreviations are added to distinguish（部分索引加上表名缩写是因为有的索引名字相同，所以加表名缩写用来区分）